sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -282,6 +282,9 @@ def _sort_buffer(self, buffer, lt):
       buffer.sort(cmp=lambda a, b: (not lt(a, b)) - (not lt(b, a)),
                   key=self._key_fn)
 
+  def display_data(self):
+    return {'n': self._n}
```

Comment:
It makes sense to include compare. If it is named use the name, otherwise something generic (like 'lambda') so that it would be clear that whether a compare argument is given or not.